ciao-controller/legacy_api.go

```diff
@@ -301,6 +325,10 @@ func legacyComputeRoutes(ctl *controller, r *mux.Router) *mux.Router {
 		legacyAPIHandler{ctl, legacyNodesSummary, true}).Methods("GET")
 	r.Handle("/v2.1/nodes/{node}/servers/detail",
 		legacyAPIHandler{ctl, legacyListNodeServers, true}).Methods("GET")
+	r.Handle("/v2.1/nodes/compute",
+		legacyAPIHandler{ctl, legacyListComputeNodes, true}).Methods("GET")
+	r.Handle("/v2.1/nodes/network",
+		legacyAPIHandler{ctl, legacyListNetworkNodes, true}).Methods("GET")
 
```

Comment:
it's a shame to add new endpoints to this legacy api - it'd really be better to add a ciao api for listing nodes - but I'm ok merging this and doing a migration later I suppose.